proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+## I. Overview
+This governance policy describes how an open source project can formally join the Foundation via the [Project Proposal Process](). It describes the [Stages]() a project may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a project to move from one stage to another. It also describes the [Annual Review Process]() through which those changes will be evaluated and made. 
+
+Benefits of the project progression process include:
+* Promotion and public visibility of work that can add value to the _$FOUNDATION_ mission and build the ingredients of a successful project
+* Deepened alignment with existing projects
+* Mentorship and support 
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to _$FOUNDATION_ legal, code of conduct and IP Policy requirements
+
+This proposal has been modified from the [CNCF process documentation](https://github.com/cncf/toc/tree/master/process).
```

Comment:
I think this is good context for now, but it probably does not belong in the document when landed.

Reply:
Agree - just wanted to credit the intellectual predecessor of this work so we have that shared context!